addon/components/polaris-data-table/cell.js

```diff
@@ -0,0 +1,192 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { htmlSafe } from '@ember/string';
+import layout from '../../templates/components/polaris-data-table/cell';
+
+export default Component.extend({
+  tagName: '',
+
+  layout,
+
+  /**
+   * @property height
+   * @type {Number}
+   * @public
+   */
+  height: null,
+
+  /**
+   * @property text
+   * @type {String|Component}
+   * @public
+   */
+  text: null,
+
+  /**
+   * @property contentType
+   * @type {String}
+   * @public
+   */
+  contentType: null,
+
+  /**
+   * @property fixed
+   * @type {boolean}
+   * @public
+   */
+  fixed: false,
+
+  /**
+   * @property truncate
+   * @type {boolean}
+   * @public
+   */
+  truncate: false,
+
+  /**
+   * @property presentational
+   * @type {boolean}
+   * @public
+   */
+  presentational: false,
```

Comment:
There are some inconsistencies in these files on whether or not there's a `@default` added - I think we want `@default` notation for any non-null attribute, yes?